users/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class User(models.Model):
+    email = models.CharField(max_length=200)
+    slack_id = models.CharField(max_length=100, unique=True)
+    user_name = models.CharField(max_length=100)
+    real_name = models.CharField(max_length=100)
+    metro_name = models.CharField(max_length=200)
+    topics_to_learn = models.CharField(max_length=1500)
+    topics_to_share = models.CharField(max_length=1500)
+    how_you_learned_about_pennyu = models.CharField(max_length=500)
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'user'
```

Comment:
This might be silly, but can we change this class and table name to "SlackUser"? It will allow us to differentiate users on slack from eventual Django users. Hopefully that will still play nicely with migrations.

Reply:
Let's talk about it. My immediate reaction is to not do this because _most_ of the data in this table corresponds to the deeper notion of a `User` - e.g. their email, topics to learn, topics to share, real name. In the future I could imagine having a `SlackUser` table or `Accounts` table or something that hold `slack_id, user_name, user_id` (the last of which being the foreign key to the user table. *Let's talk about it though. I don't completely understand your idea yet.*

I was thinking about changing the name of the table to `users_user` since that's what Django apparently wants me to do. I'd be interested on your thoughts here too.

Reply:
The reason I would like it to be at least renamed to something other than `User` is that django already provides a `User` model out of the box. https://docs.djangoproject.com/en/2.2/topics/auth/default/

Reply:
But yes, name of the table should follow Django's convention: `appname_modelname`